engine/source/mpi/r2r/spmd_r2r.F

```diff
@@ -2210,4 +2216,255 @@ SUBROUTINE SPMD_EXCH_R2R_SPHOFF(OFF_SPH_R2R,IAD_ELEM,
 
 #endif
       RETURN
-      END                   
+      END
+Chd|====================================================================
```

Comment:
@bertrand-altair 

can you put this new routine in a specific .F file?

Thanks,
Sébastien

Reply:
Done in the last commit